### PR TITLE
Resolve warnings meraki device

### PR DESCRIPTION
**Pull request by y0rune**

### CODE REVIEW

File: tests/integration/targets/meraki_device/tasks/main.yml [hunk 1]
```diff
@@ -1,246 +1,265 @@
...
+  # - ansible.builtin.assert:
   #     that:
   #       - claim_device_org.changed == true
 
   - name: Create network
-    meraki_network:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+    cisco.meraki.merak_network:
```
Comment (kbreit) [inline tests/integration/targets/meraki_device/tasks/main.yml:19]: This is a typo so the integration test didn't pass. Please resolve and I can run the tests.

Comment (y0rune) [reply tests/integration/targets/meraki_device/tasks/main.yml]: I pushed it. So you can test it ;)

**Comment by kbreit** [comment]
I'm receiving an error on this integration test due to a firmware problem. I'm going to have to troubleshoot further but I suspect the problem is on my end, not the test.

**Comment by kbreit** [comment]
@y0rune I found a bug in the playbook that I need to fix. I'll be submitting a fix shortly and that should allow this one to work but you may need to rebase on top of it.

**Pull request merged**